From 44dfd2adfc4ec0e5bc57c57c1e701a25996c8221 Mon Sep 17 00:00:00 2001
From: Po-Chuan Hsieh <sunpoet@sunpoet.net>
Date: Sat, 3 Nov 2018 18:28:11 +0800
Subject: [PATCH] Fix build with curl 7.62.0

from CHANGES:
ssl: deprecate CURLE_SSL_CACERT in favour of a unified error code
Long live CURLE_PEER_FAILED_VERIFICATION
---
 daemon/http-tx-mgr.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/daemon/http-tx-mgr.c b/daemon/http-tx-mgr.c
index 80d418ab8..4c6c65789 100644
--- a/daemon/http-tx-mgr.c
+++ b/daemon/http-tx-mgr.c
@@ -1138,7 +1138,9 @@ curl_error_to_http_task_error (int curl_error)
     case CURLE_SSL_CONNECT_ERROR:
     case CURLE_PEER_FAILED_VERIFICATION:
     case CURLE_SSL_CERTPROBLEM:
+#if LIBCURL_VERSION_NUM < 0x073e00
     case CURLE_SSL_CACERT:
+#endif
     case CURLE_SSL_CACERT_BADFILE:
     case CURLE_SSL_ISSUER_ERROR:
         return HTTP_TASK_ERR_SSL;
